tms/clickup/views.py

```diff
@@ -101,7 +104,15 @@ def create(self, request, *args, **kwargs):
             context = {"data": serializer.data, "success_message": constant.CREATE_PROJECT_SUCCESS}
             return Response(context, status=status.HTTP_201_CREATED)
 
+
 class TaskCreateView(ListCreateAPIView):
     queryset = Task.objects.all()
     serializer_class = TaskSerializer
     permission_classes = [DjangoModelPermissions]
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            context = {"data": serializer.data, "success_message": constant.CREATE_TASK_SUCCESS}
```

Comment:
you can directly pass dict in response, why take extra variable. 
return Response({"data": serializer.data, "success_message": constant.CREATE_TASK_SUCCESS}, status=status.HTTP_201_CREATED)